templates/src/serialize.c.erb

```diff
@@ -1,7 +1,4 @@
-#include "yarp/ast.h"
-#include "yarp/diagnostic.h"
-#include "yarp/parser.h"
-#include "yarp/util/yp_buffer.h"
+#include "yarp.h"
```

Comment:
Why this change?

Reply:
Otherwise:
```
serialize.c.erb: In function ‘yp_lex_serialize’:
serialize.c.erb:215:5: error: implicit declaration of function ‘yp_parser_init’; did you mean ‘yp_buffer_init’? [-Werror=implicit-function-declaration]
serialize.c.erb:223:23: error: implicit declaration of function ‘yp_parse’ [-Werror=implicit-function-declaration]
serialize.c.erb:223:23: error: initialization of ‘yp_node_t *’ {aka ‘struct yp_node *’} from ‘int’ makes pointer from integer without a cast [-Werror=int-conversion]
serialize.c.erb:237:5: error: implicit declaration of function ‘yp_node_destroy’ [-Werror=implicit-function-declaration]
serialize.c.erb:238:5: error: implicit declaration of function ‘yp_parser_free’; did you mean ‘yp_buffer_free’? [-Werror=implicit-function-declaration]
...
```
Because `yp_parser_init` is in yarp.h we have to include it, and that includes everything else.

Reply:
Got it.